test/ILLink.RoslynAnalyzer.Tests/DynamicallyAccessedMembersAnalyzerTests.cs

```diff
@@ -160,17 +160,12 @@ private static void M(Type type)
         type.GetMethod(""Bar"");
 	}
 }";
+			// The warning will be generated once dataflow is able to handle GetMethod intrinsic
 
 			// (17,9): warning IL2070: 'this' argument does not satisfy 'DynamicallyAccessedMemberTypes.PublicMethods' in call to 'System.Type.GetMethod(String)'.
 			// The parameter 'type' of method 'C.M(Type)' does not have matching annotations.
 			// The source value must declare at least the same requirements as those declared on the target location it is assigned to.
-			return VerifyDynamicallyAccessedMembersAnalyzer (TargetMethodWithAnnotations,
-				VerifyCS.Diagnostic (DiagnosticId.DynamicallyAccessedMembersMismatchParameterTargetsThisParameter)
-				.WithSpan (17, 9, 17, 30)
-				.WithArguments ("System.Type.GetMethod(String)",
-					"type",
-					"C.M(Type)",
-					"'DynamicallyAccessedMemberTypes.PublicMethods'"));
```

Comment:
Could you maybe just comment out the code - we should add it later once the intrinsics are implemented, right?